modulus/utils/capture.py

```diff
@@ -348,6 +357,8 @@ class StaticCaptureTraining(_StaticCapture):
         Number of warmup steps for cuda graphs, by default 11
     amp_type : Union[float16, bfloat16], optional
         Auto casting type for AMP, by default torch.float16
+    gradient_clip_norm : Union[float, None], optional
```

Comment:
Nit: use either `Optional[float]` or `float | None`?

Reply:
Good point, I will standard to using `Optional` instead.